### DISCUSSION TITLE
How to use Select2 in nuxt 3 ?

### DISCUSSION THREAD

**Discussion by adityafawzan** [nuxt/nuxt#17478] [Questions]
How to use Select2 in nuxt 3 ?

**Comment by wobsoriano** [comment]
`Select2 is a jQuery based replacement for select boxes`

are you sure you want to use that?

**Reply by adityafawzan** [reply]
i already tried this one https://github.com/godbasin/vue-select2/tree/npm-publish-code-for-vue3, but didnt work, any solution bro @wobsoriano ?

**Comment by TheAlexLichter** [comment]
You could probably try [this package](https://github.com/godbasin/vue-select2/tree/npm-publish-code-for-vue3) but as @wobsoriano mentioned, Select2 with its jQuery dependency is suboptimal.

**Reply by adityafawzan** [reply]
i already tried this once, but it didnt seem to work btw

![image](https://user-images.githubusercontent.com/48414316/172034488-60c4dd31-fef1-4585-ab05-eec3181fd13d.png)

**Reply by TheAlexLichter** [reply]
Indeed. Also tried it but it doesn't seem to work.

**Reply by adityafawzan** [reply]
so what now ? lol :D
